pkg/apis/extensions/v1beta1/types.go

```diff
@@ -968,27 +968,27 @@ type HostPortRange struct {
 	Max int32 `json:"max"`
 }
 
-// SELinux Context Strategy Options defines the strategy type and any options used to create the strategy.
-type SELinuxContextStrategyOptions struct {
+// SELinux Strategy Options defines the strategy type and any options used to create the strategy.
+type SELinuxStrategyOptions struct {
 	// type is the strategy that will dictate the allowable labels that may be set.
-	Type SELinuxContextStrategy `json:"type"`
-	// seLinuxOptions required to run as; required for MustRunAs
+	Type SELinuxStrategy `json:"type"`
+	// Options required to run as; required for MustRunAs
 	// More info: http://releases.k8s.io/HEAD/docs/design/security_context.md#security-context
-	SELinuxOptions *v1.SELinuxOptions `json:"seLinuxOptions,omitempty"`
```

Comment:
In this case it's a new API object and the "other" feels far enough away that doing the right thing here seems more useful than consistency.  It does break cut-paste, so I can see it both ways.  I would go with shorter myself, but I am probably not the person who will be answering emails about why it doesn't work.